jjb/mkck/mkck-trigger.yaml

```diff
@@ -282,6 +359,37 @@
             junit-attachments: true
 
 
+- job:
+    name: basic-repos
+    description: "Add basic repositories"
+    wrappers:
+        - workspace-cleanup
+        - timestamps
+        - timeout:
+            timeout: 10
+        - build-name: { name: '#$BUILD_NUMBER openSUSE Leap $VERSION' }
+    properties:
+        - groovy-label:
+            script: 'binding.getVariables().get("TARGET_NAME")'
+    builders:
+      - shell: |
+          if [ -f /etc/os-release ]; then
+            . /etc/os-release
+
+            case $ID in
+            "opensuse-leap")
+              zypper -n ar http://download.opensuse.org/distribution/leap/${VERSION_ID}/repo/oss/ oss
+              zypper -n ar http://download.opensuse.org/update/leap/${VERSION_ID}/oss/ update
```

Comment:
why you can not put this script to enable_artifacts_leap step?

Reply:
the same as it done for sles?

Reply:
(nit) I suggest to have homogeneous approach for adding repo, either move the sles adding repos to a subjob, or to have opensuse repo adding code as a step.

Reply:
Done. I renamed `enable_artifacts` to `enable_repos` and use the artifacts enablement code for sles and the repos job for opensuse.
Still we have the repo job just in the opensuse case but code-wise, it's a bit simpler I think.
Happy?